trlx/utils/modeling.py

```diff
@@ -8,6 +8,19 @@
 import torch.nn.functional as F
 import transformers
 
+try:
```

Comment:
Its an optional dependency right?

Reply:
Maybe we need a file where we just import all the optional dependencies. I worry this is going to get very gross very fast otherwise.

Reply:
Yeah, good idea! I'll make an issue to track this.